docs/_data/shapes.csv

```diff
@@ -0,0 +1,203 @@
+- Showing multi-directional data flows
+- Indicating pivot points in decision trees",Dependency: Used when multiple connections or paths need to be emphasized from a single point,,"<svg viewBox=""0 0 41 40"" fill=""none"" xmlns=""http://www.w3.org/2000/svg"">
+    <path stroke-miterlimit=""10"" d=""M30.8738 19.6872C30.6794 19.9805 30.6794 19.9805 30.8738 20.0782C32.0402 21.251 33.2065 22.3261 34.3729 23.4989C36.0252 25.1604 37.6776 26.7242 39.3299 28.3857C39.3557 28.3857 39.3804 28.396 39.3986 28.4143C39.4169 28.4326 39.4271 28.4575 39.4271 28.4834C39.3299 28.5811 39.1355 28.6789 39.0383 28.7766L34.9561 31.7087C33.9841 32.3928 32.915 33.1747 31.943 33.9566C31.8458 34.0543 31.8458 33.9566 31.7486 33.9566C30.7766 32.7837 29.8047 31.7087 28.8327 30.5358C27.472 28.9721 26.1112 27.4083 24.7505 25.7468C23.6813 24.4763 22.6121 23.3034 21.6402 22.0329L19.9879 20.0782H19.9393L14.5935 26.3332L9.24766 32.6274C8.85888 33.0183 8.56729 33.507 8.1785 33.8979C8.08131 33.9957 8.08131 33.9957 7.98411 33.8979C6.33178 32.7251 4.77664 31.5523 3.1243 30.2817C2.34673 29.793 1.56916 29.2066 0.694393 28.6202C0.694393 28.5225 0.694393 28.5225 0.597196 28.5225L0.888785 28.2293C2.54112 26.6655 4.09626 25.1018 5.7486 23.538C6.91495 22.4629 8.08131 21.2901 9.15047 20.215C9.34486 20.0195 9.34486 20.0195 9.15047 19.9218C7.98411 18.749 6.81776 17.6739 5.6514 16.5011C3.99907 14.8005 2.24953 13.2758 0.597196 11.6143L0.5 11.5166C0.597196 11.4189 0.694393 11.3211 0.791589 11.3211C2.73551 9.8551 4.67944 8.48681 6.62336 7.02079C7.10935 6.72758 7.49813 6.33664 7.98411 6.04344C8.08131 5.9457 8.08131 6.04344 8.1785 6.04344L11.5804 10.0506L16.343 15.7192L19.9393 19.9218L19.9976 19.9805L20.085 19.8827C21.4458 18.319 22.8065 16.6575 24.1673 15.0937L27.7636 10.8911L31.1654 6.88396C31.3676 6.60266 31.5954 6.3409 31.8458 6.10208C31.943 6.00434 31.943 6.00434 32.0402 6.10208C33.4981 7.17716 34.9561 8.25225 36.5112 9.32733C37.4832 10.0115 38.4551 10.6956 39.3299 11.3798H39.4271C39.5243 11.3798 39.5243 11.4775 39.4271 11.4775C39.0383 11.7707 38.7467 12.1617 38.3579 12.4549C36.0252 14.7028 33.6925 16.9507 31.457 19.1986C31.2877 19.3895 31.0912 19.5542 30.8738 19.6872Z"" fill=""#00B39F"" fill-opacity=""0.1"" stroke=""#00B39F""/>
+  </svg>"
+Trapezoid,"A four-sided shape with one pair of parallel sides, used to indicate hierarchical relationships or steps in a process.","- Representing funnels in process flows (e.g., sales funnels)
+- Indicating converging or diverging processes
+- Showing hierarchical structures with varying levels of detail",Dependency: Often used to show narrowing or widening of scope in processes,,"<svg viewBox=""0 0 40 40"" fill=""none"" xmlns=""http://www.w3.org/2000/svg"">
+<path d=""M8.5 3H32.5L39.5 37H1.5L8.5 3Z"" fill=""#00B39F"" fill-opacity=""0.1"" stroke=""#00B39F"" stroke-miterlimit=""10"" stroke-width=""1""/>
+</svg>"
+SqlJob,,,,,"<svg viewBox=""0 0 40 40"" fill=""none"" xmlns=""http://www.w3.org/2000/svg"">
```

Comment:
Comment out this shape given that it isn't included in releases yet, @sahu-01.

// @aabidsofi19